pages/index.rst

```diff
@@ -51,8 +51,8 @@ releases so far, `0.0.4`__ being the most recent one.
 
 .. __: https://hpyproject.org/blog/posts/2022/06/hpy-0.0.4-third-public-release/
 
-The Python/C API is huge and at the moment only a small fraction of it is
-available in HPy. We are experimenting by porting existing extensions to HPy
+The Python/C API is huge and at the moment the most popular functions are
+available. We are experimenting by porting existing extensions to HPy
```

Comment:
```suggestion
The Python/C API is huge and at the moment just the most popular functions are
available in HPy. We are have ported moderately sized existing extensions to HPy (e.g. kiwisolver)
and are experimenting with porting larger ones (e.g. numpy).
```

Reply:
Don't we list that information just below this paragraph? It seems redundant

Reply:
I think I can rework this section differently